invenio_rdm_records/records/mappings/v7/rdmrecords/records/record-v4.0.0.json

```diff
@@ -37,6 +37,13 @@
           "id" : {
             "type" : "keyword"
           },
+          "communities": {
+            "properties": {
+              "default": {
+                "type": "keyword"
```

Comment:
can you please add the `ids` field too? 
Did you check if there is any other field missing?